### PR TITLE
Document type resources

### PR DESCRIPTION
**Pull request by Neamar**

### CODE REVIEW

File: resources/document-types.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
+---
+title: AnyFetch document-types resources
+layout: doc
+---
+{% raw  %}
+
+You'll find here the list of default document-types and their expectations.
+Don't forget [/resources/hydraters.html](many hydraters can change the `document_type` during the hydration phase).
```
Comment (merlinND) [inline resources/document-types.md]: > Don't forget that

File: resources/document-types.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
+If you're not satisfied with the default presentation or you find some data are missing:
```
Comment (merlinND) [inline resources/document-types.md]: > some data _is_ missing

File: resources/document-types.md [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+* You can always add more `metadata`. Even though they won't be shown to the user, the document will still be returned to the user for a relevant query.
```
Comment (merlinND) [inline resources/document-types.md]: > _it_ won't be shown
> for a relevant query, as all metadata keys are used in search.

File: resources/document-types.md [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+* Full
+  - title
+  - path
+  - extension
+
+### Templates
+Snippet:
+
+```html
+<article>
```
Comment (merlinND) [inline resources/document-types.md:35]: It's too bad we have to hardcode those snippets in the doc. Would there be an easy way to get them from a config JSON somewhere to generate this `.md`?

File: resources/document-types.md [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+  <h1>{{{ title }}}</h1>
+  <code>{{{ path }}}</code>
+</article>
+```
+
+## Document
+> ID: 5252ce4ce4cfcd16f55cfa3c
+
+A document with a clean HTML representation: text, doc, xls...
```
Comment (merlinND) [inline resources/document-types.md]: Reading this, I don't get how `xls` has a clean HTML representation? Do you mean that we _can_ generate a clean HTML representation from these formats?

File: resources/document-types.md [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+
+  <img src="{{ display }}" />
+</article>
+
+```
+
+## Contact
+> ID: 5252ce4ce4cfcd16f55cfa3a
+
+A person, somewhere. (contact, client, ...)
```
Comment (merlinND) [inline resources/document-types.md]: There is an extraneous dot.
Does the "somewhere" really corresponds to geolocation data?

File: resources/document-types.md (hunk 5 above)
Comment (Neamar) [inline resources/document-types.md]: Yes

File: resources/document-types.md (hunk 4 above)
Comment (Neamar) [inline resources/document-types.md:35]: Agreed.

I'd like to add some public endpoints to the API to generate this doc in the future

Comment (merlinND) [inline resources/document-types.md:35]: Awesome!

**Pull request merged**